src/xunit.performance.api/ETWProfiler.cs

```diff
@@ -0,0 +1,300 @@
+﻿using Microsoft.Diagnostics.Tracing;
+using Microsoft.Diagnostics.Tracing.Parsers;
+using Microsoft.Diagnostics.Tracing.Session;
+using Microsoft.Xunit.Performance.Sdk;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using Xunit;
+using Xunit.Abstractions;
+using static Microsoft.Xunit.Performance.Api.Native.Windows.Kernel32;
+using static Microsoft.Xunit.Performance.Api.XunitPerformanceLogger;
+
+namespace Microsoft.Xunit.Performance.Api
+{
+    internal static class ETWProfiler
```

Comment:
We already have a version of this code elsewhere in xunit-performance.  Is it possible to use it directly (or modify it slightly)?  It would be a shame to have two ETW implementations in the same project.